plugins/awards-backend/src/service/router.ts

```diff
@@ -67,17 +67,27 @@ function buildFsAdapter(config: Config): Storage {
 
 export function getStorageClient(config: Config): Storage {
   const storageConfig = config.getConfig('awards.storage');
-  if (storageConfig.keys().length !== 1) {
+  const configs: Record<string, Config> = {};
+  storageConfig.keys().forEach(key => {
+    try {
+      configs[key] = storageConfig.getConfig(key);
+    } catch {
+      // config object isn't instantiated for this key
```

Comment:
Could log here

Reply:
this is intended though so i don't think we need to log